### PR TITLE
Postman integration doc

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206214436896259

### CODE REVIEW

File: docs/integrations/webhooks/postman.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/webhooks/postman-logo.png')} alt="Thumbnail icon" width="50"/>
+
+Postman is an API development tool that streamlines API building, testing, and modification. It simplifies the process of working with APIs for developers. Webhook available in the Postman can forward events related to changes occurred in your Postman platform to the Sumo Logic HTTP endpoint. Using these logs, users can monitor API test results, collection changes, monitor alerts, environment modifications, and custom automation workflow triggers in Sumo logic. For further details, refer [Postman Documentation](https://learning.postman.com/docs/introduction/overview/).
+
+## Event types
+
+The Sumo Logic integration for Postman ingests Postman events into Sumo Logic through an outgoing webhook available in Postman. For more information on supported events that are ingested through the Postman webhook, refer [Postman documentation](https://learning.postman.com/docs/integrations/webhooks/).
```
Comment (jpipkin1) [inline docs/integrations/webhooks/postman.md:14]:
```suggestion
The Sumo Logic integration for Postman ingests Postman events into Sumo Logic through an outgoing webhook available in Postman. For more information on supported events that are ingested through the Postman webhook, refer to [Postman documentation](https://learning.postman.com/docs/integrations/webhooks/).
```

File: docs/integrations/webhooks/postman.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## Setup
+
+### Source configuration
+
+Follow the below steps to configure the Hosted Collector and receive Postman events:
+
+1. In the Sumo Logic portal, create a new [Hosted Collector](https://help.sumologic.com/docs/send-data/hosted-collectors/configure-hosted-collector/) or use an existing one. Then add a [HTTP Logs and Metrics Source](https://help.sumologic.com/docs/send-data/hosted-collectors/http-source/logs-metrics/#configure-an-httplogs-and-metrics-source).
```
Comment (jpipkin1) [inline docs/integrations/webhooks/postman.md:22]:
```suggestion
1. In the Sumo Logic portal, create a new [Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector/) or use an existing one. Then add an [HTTP Logs and Metrics Source](/docs/send-data/hosted-collectors/http-source/logs-metrics#configure-an-httplogs-and-metrics-source). 
```

File: docs/integrations/webhooks/postman.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+2. Configure **Source Category** in the HTTP Source. For example, configure as `webhook/postman` for the Postman app.
+
+3. Copy and save the endpoint URL of source.
+
+### Vendor configuration
+
+Configure the Webhook Integration in Postman to connect to your Sumo Logic HTTP Source. Once configured, it will be triggered each time the events occur within your Postman account.
+
+Follow the below steps to configure the Postman Webhook:
```
Comment (jpipkin1) [inline docs/integrations/webhooks/postman.md:32]:
```suggestion
Follow the below steps to configure the Postman webhook:
```

File: docs/integrations/webhooks/postman.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+1. Sign in to your [Postman account](https://identity.getpostman.com/login?continue=https%3A%2F%2Fgo.postman.co%2Fbuild).
+1. On the home page, click on the **Integrations** tab from the left sidebar.
+1. Search and select **Webhooks** from the list of integrations.
+1. The integration page has choices for each type of custom webhook. Select **Add Integration** next to a webhook type to configure your integration. The **Add integration** form appears.
+1. Enter Webhook form data as follows:
```
Comment (jpipkin1) [inline docs/integrations/webhooks/postman.md:38]:
```suggestion
1. Enter webhook form data as follows:
```

File: docs/integrations/webhooks/postman.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    1. For **Post monitoring results** as the webhook type,
+        - **Choose workspace**. Select the workspace to which your collection belongs.
+        - **Choose Monitor**. Select the monitor you wish to send results for.
+    1. Click **Add Integration**.
+1. Verify the ingestion of Postman events in Sumo Logic by executing the following query on Sumo Logic's search panel.
+
+    `_sourcecategory=webhook/postman`
+
+:::note
+- For detailed information about webhook creation, refer [Postman Documentation](https://learning.postman.com/docs/integrations/webhooks/).
```
Comment (jpipkin1) [inline docs/integrations/webhooks/postman.md:53]:
```suggestion
- For detailed information about webhook creation, refer to [Postman documentation](https://learning.postman.com/docs/integrations/webhooks/).
```

File: docs/integrations/webhooks/postman.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+- For support, contact [Postman support center](https://support.postman.com/hc/en-us).
```
Comment (jpipkin1) [inline docs/integrations/webhooks/postman.md:54]:
```suggestion
- For support, contact the [Postman support center](https://support.postman.com/hc/en-us). 
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@JV0812, I mostly have minor updates.

**Comment by JV0812** [comment]
@jpipkin1 Thanks for your review John. I have fixed your comments.

**Review by jpipkin1 — APPROVED** [review]

**Comment by jpipkin1** [comment]
Approved, @JV0812.

I just now noticed you need to create a hub page for "Webhooks" in the left nav.  But in my opinion it's not essential. You can do that later if you want.

**Comment by JV0812** [comment]
Yes John. I will create it later. 
There will be around 100 webhooks apps docs added to this folder, so I thought I can create the webhooks hub page once all the apps are added. By this we can reduce the continuous update of this page whenever a new app is added.

File: docs/integrations/webhooks/postman.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+### Source configuration
+
+Follow the below steps to configure the Hosted Collector and receive Postman events:
+
+1. In the Sumo Logic portal, create a new [Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector/) or use an existing one. Then add an [HTTP Logs and Metrics Source](/docs/send-data/hosted-collectors/http-source/logs-metrics#configure-an-httplogs-and-metrics-source). 
+
+2. Configure **Source Category** in the HTTP Source. For example, configure as `webhook/postman` for the Postman app.
+
+3. Copy and save the endpoint URL of source.
```
Comment (kimsauce) [inline docs/integrations/webhooks/postman.md:26]:
```suggestion
2. Configure **Source Category** in the HTTP Source. For example, configure as `webhook/postman` for the Postman app.
3. Copy and save the endpoint URL of source.
```

**Review by kimsauce — COMMENTED** [review]
@JV0812 I've swapped out image with hi-res version

File: sidebars.ts [hunk 8]
```diff
@@ -2230,6 +2230,15 @@ integrations: [
           'integrations/sumo-apps/security-analytics',
         ],
       },
+      {
+        type: 'category',
+        label: 'Webhooks',
+        collapsible: true,
+        collapsed: true,
+        items: [
+          'integrations/webhooks/postman',
```
Comment (kimsauce) [inline sidebars.ts:2239]:
```suggestion
          'integrations/webhooks/buddy',
          'integrations/webhooks/postman',
```

**Comment by jpipkin1** [comment]
@kimsauce, your sidebar change is causing the build to fail ([details](https://github.com/SumoLogic/sumologic-documentation/actions/runs/5835987239/job/15828632478?pr=2924)):
> Error:  Error: Invalid sidebar file at "sidebars.ts".

File: sidebars.ts [hunk 9]
```diff
@@ -2230,6 +2230,16 @@ integrations: [
           'integrations/sumo-apps/security-analytics',
         ],
       },
+      {
+        type: 'category',
+        label: 'Webhooks',
+        collapsible: true,
+        collapsed: true,
+        items: [
+          'integrations/webhooks/buddy',
```
Comment (kimsauce) [inline sidebars.ts:2239]:
```suggestion
```

**Comment by JV0812** [comment]
The intent of this PR is covered through a consolidated PR.

**Pull request closed without merging**